libraries/botframework-connector/src/customConnectorApi/conversationsApi.ts

```diff
@@ -74,65 +75,67 @@ export class ConversationsApi {
         return `${ await tokenGenerator.getToken(true) }`;
     }
 
-
+    private async deserializeResponse<T>(url, requestOptions, type): Promise<T> {
+        return new Promise<T>((resolve, reject) => {
+            fetch(url, requestOptions).then(response => {         
+                let httpResponse: http.IncomingMessage = response;
+                
+                if (response.status &&  response.status >= HttpStatus.OK && response.status < HttpStatus.MULTIPLE_CHOICES) { 
```

Comment:
There are 2 whitespaces between the `&&` and `response.status`. Delete one of them :triggered: